hw/lint/common.core

```diff
@@ -7,7 +7,7 @@ description: "Common waivers"
 filesets:
   files_verilator:
     files:
-      - tools/verilator/common.vlt
+      - tools/verilatorlint/common.vlt
```

Comment:
Do we really need this rename? The lint waivers are for Verilator, there's no separate "Verilator lint" tool (in contrast with Verible). These waivers are equally needed for compiling a simulation with Verilator.

Reply:
Indeed, let me revert that. I think I made that change primarily since dvsim chooses the flow class based on the tool right now, but this is something we need to address differently anyways, such that you can use the same tool (verilator), but with different flows (lint, simulation).

Reply:
ok this has been reverted to `verilator`